lib/internal/bootstrap/node.js

```diff
@@ -319,6 +319,27 @@ ObjectDefineProperty(process, 'features', {
 }
 
 const { emitWarning, emitWarningSync } = require('internal/process/warning');
+const { getOptionValue } = require('internal/options');
+
+let kTypeStrippingMode;
+ObjectDefineProperty(process.features, 'typescript', {
+  __proto__: null,
+  get() {
+    if (kTypeStrippingMode === undefined) {
```

Comment:
```suggestion
    if (kTypeStrippingMode === undefined) {
```
Also if you set to empty string it would not be equal to undefined and it will set it again

Reply:
What do you mean? The point was that an empty does not equal undefined, so it won't set it again.

Regardless, I changed it per your last review.